app/models/agents/rss_agent.rb

```diff
@@ -53,9 +49,26 @@ def default_options
       Events look like:
 
           {
+            "feed": {
```

Comment:
Downstream agents may want to use it; for instance, a DataOutputAgent could have `template.title` set to `{{events.first.feed.title}} full text feed` to follow DRY.